src/show.jl

```diff
@@ -63,24 +62,19 @@ for ArbT in (Arf, Arb, Acb, Mag)
     end
 end
 
-for ArbT in (Arf, Arb, Mag)
-    jlf = :_load_string!
-    arbf = Symbol(cprefix(ArbT), :_load_str)
+for ArbT in (Mag, Arf, Arb)
     @eval begin
-        function $(jlf)(x::$ArbT, str::AbstractString)
-            res = ccall(@libarb($arbf), Cint, (Ref{$ArbT}, Cstring), x, str)
-            iszero(res) || throw(ArgumentError("arblib could not load_str $str as "*$(string(ArbT))))
+        function load_string!(x::$ArbT, str::AbstractString)
+            res = load!(x, str)
+            iszero(res) || throw(ArgumentError("could not load $str as " * $(string(ArbT))))
             return x
         end
-    end
 
-    jlf = :_dump_string
-    arbf = Symbol(cprefix(ArbT), :_dump_str)
-    @eval begin
-        function $(jlf)(x::$ArbT)
-            char_ptr = ccall(@libarb($arbf), Cstring, (Ref{$ArbT},), x)
+        function dump_string(x::$ArbT)
+            char_ptr = dump(x)
             str = unsafe_string(char_ptr)
-            ccall((:flint_free, libflint), Cvoid, (Cstring,), char_ptr)
+            # TODO: Fix this!
+            #ccall((:flint_free, LoadFlint.libflint), Cvoid, (Cstring,), char_ptr)
```

Comment:
This is the part that remains to fix.

Reply:
oh, yes, I actually forgot to finish `LoadFlint` stuff, these times are crazy for me.
We need to
1. delete references to `flint` from `deps.jl`
2. after `using LoadFlint` define `const libflint = LoadFlint.libflint`
3. leave the ccall as it is (or define `@libflint` macro)
should I push here?

Reply:
Feel free to push here!